doc/sphinx/conf.py

```diff
@@ -242,7 +242,6 @@ def setup(app):
 
 # Add any paths that contain custom themes here, relative to this directory.
 import sphinx_rtd_theme
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

Comment:
The result looks good, but I'm trying to understand what the code change does. You basically remove this dependency and instead vendor this CSS code and adjust it? It was not possible to add a CSS rule to override the previous behavior? The vendored CSS code is pretty unreadable because it is minimized. We might also need to specify its origin and its license.

Reply:
this was causing a warning that was turned into an error in dune. The warning said that this was deprecated and could be safely removed.

Reply:
Thanks for the explanation of this part, but this does not answer the rest of the question (where does all this CSS code come from and why is it minimized), so leaving the discussion unresolved.

Reply:
We made a modification of the sphinx theme, which is distributed minimized (your editor has a beautifier probably). I changed the patch to rather simply customize the theme.

Reply:
Yes, it's no longer needed afaiu